### ISSUE TITLE
Display object permission in group and user admin

### ISSUE THREAD

**Issue by ziima** [django-guardian/django-guardian#739]
Object permissions are quite easily integrated, but since they are stored separately from `auth` permissions those permissions are displayed nowhere else in the administration interface. They're especially missing in the views for groups and users, where permissions are displayed. I suggest to provide a support for displaying (possibly editing) object permissions in group and user administration.

**Comment by thclark** [comment]
Yup, the guarded model admin allows you to do this, so does the equivalent integration for django-unfold.
